pycosep/community_separability.py

```diff
@@ -327,8 +329,8 @@ def _randomize_communities(communities, total_permutations):
     total_communities = len(communities)
 
     for ix in range(total_permutations):
-        np.random.seed(ix)
-        positions = np.random.permutation(total_communities)
+        np.random.seed(ix+1)            # to equal the Matlab random permutation indices
+        positions = np.argsort(np.random.random((1,total_communities)))[0]
```

Comment:
Why do you need this? 🤔

Reply:
What is the benefit of using this approach versus `np.random.permutation`?

Reply:
Ahh I think I got it. The Fisher-Yates shuffle in NumPy does not explicitly generate and sort a list of random numbers as MATLAB does, and your approach seems to do it the same way. However, I am a bit concerned about performance.